### ISSUE TITLE
Question about training loss

### ISSUE THREAD

**Issue by Mordoal** [ZhouCX117/UAED#18]
I'm training UAED on my own dataset, which one image only has one label. And i've modified lb_mean and lb_std in data_loader_one_random_uncert.py, that is label.mean and label.std. However, the bce_loss and std_loss only decay till front a few epoch. finally, bce_loss remains at round 4000, std_loss remains at 40000. may i have ur suggestions about how i can decrease the loss?

**Comment by ZhouCX117** [comment]
What is your image size? I check the log for pretraining on the VOC dataset, and the bce loss seems larger than the std loss. You can refer it.
[Adam-0-log.txt](https://github.com/ZhouCX117/UAED/files/13536061/Adam-0-log.txt)

**Comment by ZhouCX117** [comment]
Does the label value range from 0 to 1? What do you mean torch.relu() for the final outputs? Replace torch.sigmoid() or nn.Softplus() for the std output?
